### ISSUE TITLE
Cannot start transmission in libreelec 8, odroid C2

### ISSUE THREAD

**Issue by antoinegomez** [thoradia/LibreELEC.tv#26]
Hello,

I tried to download transmission with thoradia repo on libreelec 8 on my Odroid C2.

Libreelec version is: LibreELEC-Odroid_C2.aarch64-8.0.2.img.gz

I downloaded service.thoradia-8.0.2.zip and then fetch the addon from the repo on Kodi.

Version is 8.0.6.

I guess the problem is that the binary is not compatible with my libreelec version.

Here is the log I have:

```# sh -exec "sh /storage/.kodi/addons/service.transmission/bin/transmission-daemon"
+ sh /storage/.kodi/addons/service.transmission/bin/transmission-daemon
/storage/.kodi/addons/service.transmission/bin/transmission-daemon: line 1: syntax error: unexpected word (expecting ")")```

```service.transmission.service - transmission
   Loaded: loaded (/storage/.kodi/addons/service.transmission/system.d/service.transmission.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Mon 2017-08-28 11:39:17 CEST; 1 months 28 days ago
  Process: 790 ExecStart=/bin/sh /storage/.kodi/addons/service.transmission/bin/transmission.start (code=exited, status=127)

Aug 28 11:39:16 KodiStrassen systemd[1]: Failed to start transmission.
Aug 28 11:39:16 KodiStrassen systemd[1]: service.transmission.service: Unit entered failed state.
Aug 28 11:39:16 KodiStrassen systemd[1]: service.transmission.service: Failed with result 'exit-code'.
Aug 28 11:39:17 KodiStrassen systemd[1]: service.transmission.service: Service hold-off time over, scheduling restart.
Aug 28 11:39:17 KodiStrassen systemd[1]: Stopped transmission.
Aug 28 11:39:17 KodiStrassen systemd[1]: service.transmission.service: Start request repeated too quickly.
Aug 28 11:39:17 KodiStrassen systemd[1]: Failed to start transmission.
Aug 28 11:39:17 KodiStrassen systemd[1]: service.transmission.service: Unit entered failed state.
Aug 28 11:39:17 KodiStrassen systemd[1]: service.transmission.service: Failed with result 'exit-code'.```

Thanks for the help!

**Comment by thoradia** [comment]
Installation instructions are here: https://github.com/thoradia/thoradia
Bootstrap add-on is here: https://raw.githubusercontent.com/thoradia/thoradia/master/service.thoradia.zip

Install the bootstrap add-on, restart Kodi, update add-ons, enjoy...
Let me know if you have problems

**Comment by antoinegomez** [comment]
```
ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.RuntimeError'>
                                            Error Contents: Unknown addon id 'repository.libreelec.tv'.
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/service.thoradia/default.py", line 26, in <module>
                                                release = getTag(getTree(getXML('repository.libreelec.tv')), 'datadir').text.strip('/').split('/')[-3:]
                                              File "/storage/.kodi/addons/service.thoradia/default.py", line 18, in getXML
                                                return os.path.join(xbmcaddon.Addon(*id).getAddonInfo('path'), 'addon.xml')
                                            RuntimeError: Unknown addon id 'repository.libreelec.tv'.
                                            -->End of Python script error report<--
```

I think it's because it's not meant for LibreElec version 8.0.2.

**Comment by antoinegomez** [comment]
After a clean install it's working fine. Sorry for the trouble :)

**Comment by avinson** [comment]
just a note for others finding this.. solution that worked for me:
- uninstall thoradia repo (but not the individual addons)
- install new bootstrap zip file using "install from zipfile" in kodi
- throadia addons should now update to the new LE 8.2 versions

**Comment by TheUglyOne** [comment]
Another note for people finding this:

After reinstalling from the bootstrap zip, it was showing Thoradia Add-Ons as version 0.0.0 in Add-ons > My add-ons > Services. If I tried to update any of the services from this menu, it would say they were already up-to-date (even though they were version 8.1.x).

To get it to work, I had to browse to Add-ons > Add-on browser (the box icon at the top left) > Install from repository > Thoradia Add-ons > Services. The services I already had installed had an 'X' icon next to them until I clicked on them to update them. Now they're at v8.2.x and working well.
